app/services/ai_backend/open_ai.rb

```diff
@@ -76,6 +79,7 @@ def stream_handler(&chunk_received_handler)
   end
 
   def system_message
+    return [] if @assistant.openai_o1?
```

Comment:
... because system messages aren't supported?

Reply:
Correct. We received a 400 Bad Request stating that messages with the "system" role were unsupported. This fixes it and I chose not to try to push the system message into a regular user message because it didn't feel quite right.